components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -168,14 +181,27 @@ export default function ProjectSettingsView() {
                 />
                 {enablePrebuilds && (
                     <>
-                        <InputField label="Workspace machine type" disabled={!enablePrebuilds}>
-                            <div className="max-w-md">
-                                <SelectWorkspaceClassComponent
-                                    disabled={!enablePrebuilds}
-                                    selectedWorkspaceClass={project.settings?.workspaceClasses?.prebuild}
-                                    onSelectionChange={setWorkspaceClassForPrebuild}
-                                />
-                            </div>
+                        <SelectInputField
+                            disabled={!enablePrebuilds}
+                            label="Build branches"
+                            value={prebuildBranchStrategy}
+                            containerClassName="max-w-md ml-6 text-sm"
+                            onChange={(val) => setPrebuildBranchStrategy(val as ProjectSettings.PrebuildBranchStrategy)}
+                        >
+                            <option value="defaultBranch">Default branch (e.g. main)</option>
```

Comment:
**issue(non-blocking)**: Not sure if we need the example branch name here, but minor—we can remove at any time later.